daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -42,6 +44,9 @@ func NewRotateFileWriter(logPath string, capacity int64, maxFiles int) (*RotateF
 //WriteLog write log message to File
 func (w *RotateFileWriter) Write(message []byte) (int, error) {
 	w.mu.Lock()
+	if w.closed {
+		return -1, errors.New("cannot write because the output file was closed")
```

Comment:
Ah, my mistake - I drafted this up too quickly to check if this was what @cpuguy83 had in mind; I'll update